app/controllers/restaurant_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RestaurantController < ApplicationController
+
+  def show
+    service = RevItUpService.new(current_user.lat, current_user.lng, meters_to_miles, params["price"])
+    @restaurant = RevItUpFacade.new(service.restaurant)
+  end
+
+  def meters_to_miles
+    (params["distance"].to_f * 1609)
```

Comment:
Now all distances from our drop down will be able to be converted from miles to meters.